### DISCUSSION TITLE
practice-exam/practice-exam-22

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#84] [Comments]
# practice-exam/practice-exam-22

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-22.html

**Comment by jasonwurzel** [comment]
In Question 5: 

Which AWS service or feature allows the user to manager cross-region application traffic?
A. Amazon AppStream 2.0
B. Amazon VPC
C. Elastic Load Balancer
D. Amazon Route 53
Answer
Correct Answer: A

You have A (AppStream) as the answer. Isn't the correct answer D?
- AppStream is used to stream applications to your desktop
- Route 53 can be used to manage network traffic globally

so, I'd say D would be the right one...
